servicetalk-http-router-jersey/src/main/java/io/servicetalk/http/router/jersey/JerseyRouteExecutionStrategy.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright © 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.http.router.jersey;
+
+import io.servicetalk.concurrent.api.Executor;
+import io.servicetalk.concurrent.api.Publisher;
+import io.servicetalk.concurrent.api.Single;
+import io.servicetalk.http.api.HttpExecutionStrategy;
+
+import javax.annotation.Nullable;
+
+import static java.util.Objects.requireNonNull;
+
+class JerseyRouteExecutionStrategy implements HttpExecutionStrategy {
+
+    private final HttpExecutionStrategy delegate;
+    @Nullable
```

Comment:
is this necessary? looks like the usage of this class prevents `@Nullable`. if it is intended then update the accessor method to have this annotation too and protect against `null` for usages (e.g. in equals, hashCode).

Reply:
It does not need to allow `@Nullable`, removed.